hw0/README.rst

```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```

Comment:
Please do not include changes of other branch.  It seems that this PR did not branch from `hw1`.

Reply:
Thank you. I have changed the wrong commit.

Reply:
No you didn't.  See https://github.com/yungyuc/nsdhw_21au/pull/20/commits .  The unwanted change is still in your branch.

Reply:
Thanks. I'll close this PR and re-branch it.